src/entries/popup/components/CoinIcon/CoinIcon.tsx

```diff
@@ -282,12 +282,13 @@ export const NFTIcon = ({
         width={size}
         borderRadius={nftRadiusBySize[size]}
         onError={() => setBadSrc(true)}
+        maxHeight={size}
       />
-      <Box position="absolute" bottom="0" style={{ zIndex: 2, left: '-6px' }}>
-        {badge && chainId !== ChainId.mainnet && (
+      {badge && chainId !== ChainId.mainnet && (
```

Comment:
also moved this to not render this empty box for the NFT components

Reply:
nicee